Python/initconfig.c

```diff
@@ -312,7 +312,8 @@ The following implementation-specific options are available:\n\
 "-X gil=[0|1]: enable (1) or disable (0) the GIL; also PYTHON_GIL\n"
 #endif
 "\
--X importtime: show how long each import takes; also PYTHONPROFILEIMPORTTIME\n\
+-X importtime[=2]: show how long each import takes; use -X importtime=2 to\
+         log imports of already-loaded modules; also PYTHONPROFILEIMPORTTIME\n\
```

Comment:
This is missing a newline:
```suggestion
-X importtime[=2]: show how long each import takes; use -X importtime=2 to\n\
         log imports of already-loaded modules; also PYTHONPROFILEIMPORTTIME\n\
```
```console
$ python3.14 --help-all
...
-X frozen_modules=[on|off]: whether to use frozen modules; the default is "on"
         for installed Python and "off" for a local build;
         also PYTHON_FROZEN_MODULES
-X importtime[=2]: show how long each import takes; use -X importtime=2 to         log imports of already-loaded modules; also PYTHONPROFILEIMPORTTIME
-X int_max_str_digits=N: limit the size of int<->str conversions;
         0 disables the limit; also PYTHONINTMAXSTRDIGITS
-X no_debug_ranges: don't include extra location information in code objects;
         also PYTHONNODEBUGRANGES
-X perf: support the Linux "perf" profiler; also PYTHONPERFSUPPORT=1
...
```
Please see PR https://github.com/python/cpython/pull/136391 for a fix.